src/Queue/QueueServiceProvider.php

```diff
@@ -64,7 +63,8 @@ public function register()
             return new Worker(
                 new HackyManagerForWorker($app[Factory::class]),
                 $app['events'],
-                $app[ExceptionHandling::class]
+                $app[ExceptionHandling::class],
+                function () { return false; }
```

Comment:
What is this parameter for? Would be nice to see a comment at least

Reply:
Made it as a separate PR, and I've also commented what that does. https://github.com/flarum/core/pull/2357